packages/office-ui-fabric-react/src/components/Layer/Layer.types.ts

```diff
@@ -48,6 +48,13 @@ export interface ILayerProps extends React.HTMLAttributes<HTMLDivElement | Layer
    * content in a fixed position element rendered at the end of the document.
    */
   hostId?: string;
+
+  /**
+   * When enabled, Layer allows events to bubble up from Layer content.
+   * Traditionally Layer has not had this behavior. This prop preserves backwards compatibility by
+   * default while allowing users to opt in to the new event bubbling functionality.
+   */
+  eventBubblingEnabled?: boolean;
```

Comment:
Will this limit us going forward? Should we have a new Layer that does this by default and that we recommend people to use?

Reply:
That's a good question. In many use cases (Modals, Panels, Dialogs, etc.) I don't think you'll ever want event bubbling behavior. While portals more accurately portray virtual hierarchy, I don't think it's how people will ever want these common use cases to behave.

Reply:
I think I agree here. 

The spirit of a layer is that you are rendering on a higher elevation than the default body. It really is an attached layer, like photoshop layers, which simply overlay and do not interact.

There are cases where you need to know the originating dom heirarchy, and we have virtual parents for that. But I've never run into a case where events should bubble into or out of layers. That would kind of break the spirit.

Now I have seen web sites which use DOM eventing to raise all sorts of custom action events, so that they can be mediated through the root dom. Kind of like redux and central state, but with respect to DOM structure to provide a framework agnostic but tree sensitive data flow. In that case? Sure I could see a need to propagate events.